crates/forge/src/fuzzer.rs

```diff
@@ -0,0 +1,33 @@
+use cairo_felt::Felt252;
+use num_bigint::{BigUint, RandBigInt};
+use num_traits::Zero;
+use rand::rngs::StdRng;
+use rand::{thread_rng, RngCore, SeedableRng};
+
+pub struct Random {
+    rng: Box<dyn RngCore>,
```

Comment:
I guess we could change the type to [ThreadRng](https://docs.rs/rand/latest/rand/rngs/struct.ThreadRng.html#) since that's the only type we use when creating rng field

Reply:
Actually, I'll change it